Dockerfile

```diff
@@ -36,13 +34,20 @@ RUN set -x && \
     rm -rf "$GNUPGHOME" /usr/local/bin/gosu.asc && \
     chmod +x /usr/local/bin/gosu && \
     gosu nobody true && \
+    curl -L -O "https://github.com/containernetworking/plugins/releases/download/${CNI_PLUGINS_VERSION}/cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz" && \
+    curl -L -O "https://github.com/containernetworking/plugins/releases/download/${CNI_PLUGINS_VERSION}/cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz.asc" && \
+    export GNUPGHOME="$(mktemp -d)" && \
+    gpg --keyserver pgp.mit.edu --keyserver keyserver.pgp.com --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 5B1053CE38EA2E0FEB956C0595BC5E3F3F1B2C87 && \
+    gpg --batch --verify cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz.asc cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz && \
+    mkdir -p /opt/cni/bin && \
+    tar xf cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz -C /opt/cni/bin && \
+    rm -rf "$GNUPGHOME" cni-plugins-linux-amd64-${CNI_PLUGINS_VERSION}.tgz* && \
```

Comment:
Can you split this out in a dedicated `RUN` command, to run between the `gosu` installation one and the Nomad one?

It's a small optimization but I feel the CNI plugins will evolve faster than `gosu` (!) and slower than Nomad so having a dedicated layer would make sense.

Reply:
Actually, I'll do it myself before merging.

Reply:
oki, https://github.com/greut/docker-nomad/commit/8ec6a8eaa7acf097f5f0683708d50e597edf4c1c